src/widgets/vitals/vitals-form.component.tsx

```diff
@@ -749,7 +748,6 @@ export function VitalsForm(props: vitalsFormProp) {
               ? styles.buttonStyles
               : `${styles.buttonStyles} ${styles.buttonStylesBorder}`
           }
-          style={{ position: "absolute", bottom: 0 }}
```

Comment:
Per @donaldkibet , this is suppose to be absolute as the button is supposed to be at the bottom. Does this achieve the right design?